editor/src/components/canvas/canvas-strategies/strategies/flow-reorder-strategy.spec.browser2.tsx

```diff
@@ -217,6 +286,8 @@ describe('Flow Reorder Strategy (Mixed Display Type)', () => {
     ])
 
     await renderResult.getDispatchFollowUpActionsFinished()
+
+    await wait(10000)
```

Comment:
Is this necessary? As it usually indicates we're not `await`ing something somewhere.

Reply:
I forgot to delete this after debugging tests, I removed them in https://github.com/concrete-utopia/utopia/pull/3302/commits/a01cb2d89c9dd09f7b47016a21b4c70d2256b64b